website/content/docs/connect/cluster-peering/usage/create-sameness-groups.mdx

```diff
@@ -6,23 +6,19 @@ description: |-
 
 # Create sameness groups
 
-This topic describes how to designate groups of services as functionally identical in your network. Sameness groups are the preferred failover strategy for deployments with cluster peering connections. For more information about sameness groups and failover, refer to [Failover overview](/consul/docs/connect/failover).
+This topic describes how to create a sameness group, which designates a set of admin partitions as functionally identical in your network. Adding an admin partition to a sameness group enables Consul to recognize services registered to different clusters as instances of the same service when they share a name and namespace.
 
-<Warning>
-
-Sameness groups are a beta feature for all Consul v1.16.x releases. Functionality is subject to change. You should never use the beta release in secure environments or production scenarios. Features in beta may experience performance issues, scaling issues, and limited support.
-
-</Warning>
+For information about configuring a failover strategy using sameness groups, refer to [Failover with sameness groups](/consul/docs/connect/failover/sameness).
 
 ## Workflow
 
-Sameness groups are a user-defined set of partitions that Consul uses to identify services in different administrative partitions with the same name and namespace as being the same service. One of the use cases for sameness groups is to create a blanket failover policy for deployments with cluster peering connections.
+Sameness groups are a user-defined set of partitions with established cluster peering connections between each other. These partitions should have identical configurations, including configuration entries for service and proxy defaults.
```

Comment:
sameness groups do not need to be peered together, they can exist on the same cluster